From bc0d3d5ac25ab8ee3fbdb94886894e321b99be1b Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Tue, 26 Nov 2013 17:21:13 +0000
Subject: [PATCH 053/351] ASoC: florida: Enable DRC2 Signal Detect output

The DRC2 Signal Detect output needs to be enabled whilst an output is
being taken from the DRC. Enable this at the same point as we enable the
interrupt output from the DRC.

Change-Id: I4fedd96ac07f18cbd639d61020a9fc21d2e01447
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Signed-off-by: Nikesh Oswal <Nikesh.Oswal@wolfsonmicro.com>
---
 sound/soc/codecs/florida.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/codecs/florida.c b/sound/soc/codecs/florida.c
index 03a6db3..d7cea71 100644
--- a/sound/soc/codecs/florida.c
+++ b/sound/soc/codecs/florida.c
@@ -1930,6 +1930,7 @@ static int florida_codec_probe(struct snd_soc_codec *codec)
 		return ret;
 	}
 
+	snd_soc_dapm_enable_pin(&codec->dapm, "DRC2 Signal Activity");
 	ret = regmap_update_bits(arizona->regmap, ARIZONA_IRQ2_STATUS_3_MASK,
 				 ARIZONA_IM_DRC2_SIG_DET_EINT2,
 				 ARIZONA_IM_DRC2_SIG_DET_EINT2);
-- 
1.7.9.5

